docs/docs/reference/fast-refresh.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Fast Refresh
```

Comment:
I think it makes sense to move this doc under the "local development" folder inside of Reference. What do you think?

Reply:
@meganesu The folder didn't exist so I created it now